recipes-tedge/tedge-bootstrap/tedge-bootstrap/tedge-bootstrap

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+set -e
+
+RAN_MARKER=/etc/tedge/.bootstrapped
+
+log() {
+    echo "$@" >&2
+}
+
+set_hostname() {
+    host_name="$1"
+    if [ "$(cat /etc/hostname)" = "$host_name" ]; then
+        echo "Host name is already set"
+        return
+    fi
+    echo "$host_name" | tee /etc/hostname
+    sed -i -E 's/^127.0.1.1.*/127.0.1.1\t'"$host_name"'/' /etc/hosts
+    hostnamectl set-hostname "$host_name"
```

Comment:
question: does this `tedge-bootstrap` service only run when we're using systemd, i.e. boostrapping doesn't work with sysvinit?

Reply:
yes currently as we just haven't created a sysvinit service. Though I would do this in another PR